mmv1/third_party/terraform/website/docs/r/sql_user.html.markdown

```diff
@@ -130,6 +130,8 @@ The following arguments are supported:
 
     Possible values are: `ABANDON`.
 
+* `password_wo_version` - (Optional) The version of the password_wo.
```

Comment:
The WO field docs are all fairly terse. Should we link out to guides explaining the version convention and how to update WO values?

Reply:
description contains link to updating-write-only section now